contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -51,32 +52,40 @@ const (
 	Deleted  = FlagType("deleted")
 )
 
+var defaultRoles = []string{"*"}
+
 // A struct that describes a pod task.
 type T struct {
 	ID          string
 	Pod         api.Pod
-	Spec        Spec
+	Spec        *Spec
```

Comment:
it looks like clones will share this field. is that expected? If so, it should be documented in Clone().